desc/equilibrium/equilibrium.py

```diff
@@ -1339,55 +1345,21 @@ def _get_rtz_grid(
             self, radial, poloidal, toroidal, coordinates, period, jitable, **kwargs
         )
 
-    def compute_theta_coords(
-        self, flux_coords, L_lmn=None, tol=1e-6, maxiter=20, full_output=False, **kwargs
+    @staticmethod
+    def _map_clebsch_coordinates(
```

Comment:
does this need to be a method here? I'd vote for just using the function directly where needed

Reply:
It has to be an object method to avoid circular imports.